Makefile.am

```diff
@@ -1,8 +1,10 @@
+ACLOCAL_AMFLAGS = -I m4
+
 SUBDIRS = sources sources/glue doc
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = webkitgtk-sharp-3.0.pc
 
-EXTRA_DIST = webkitgtk-sharp-3.0.pc.in out/webkitgtk-sharp.dll.config
```

Comment:
Well, I'm not sure. Let me explain this: Thanh Tung Nguyen has developed a Debian packaging for webkitgtk-sharp but has not uploaded it to Debian. I was going to update SparkleShare in Debian (we use it at @Collabora), and it uses this library. I found the PPA by Nguyen, updated it and fixed a couple of bugs in his packaging and I'm going to upload it to Debian.

I tried to review the patches, I dropped some changes not relevant upstream, but I have overlooked this particular change. This file still ends up installed, so I _suppose_ something else installs it — but I don't have the clear understanding what exactly does that. I will try to ask the guy what was his intent when he wrote this.

If you also can't figure this out, please ignore this particular change for now.